movai_core_shared/core/zmq_client.py

```diff
@@ -129,69 +116,51 @@ def _extract_reponse(self, buffer: bytes):
             )
             return {}
 
-    def recieve(self) -> dict:
```

Comment:
why did you removed this function ?
This will break metrics api and some other functionatlities

Reply:
it will not break 
Because nothing is using zmq_client directly 
only throught message_client that stayed the same